ui/package.json

```diff
@@ -5,8 +5,8 @@
     "ajv": "^8.12.0",
     "ajv-formats": "^2.1.1",
     "ajv-formats-draft2019": "^1.6.1",
-    "antd": "^5.18.0",
-    "axios": "^1.6.1",
+    "antd": "^5.20.4",
+    "axios": "^1.7.4",
```

Comment:
We should update all libraries, at least to their newest compatible version, before the release.

Reply:
updated but without eslint and vite-tsconfig-paths packages because that packages make an issues, as I understand @iden3/eslint-config-react-ts should be updated for using a new eslint version. vite-tsconfig-paths also have an issue, and information that I found that fsevents that rollup used should be updated, but it's not our direct dependence

Reply:
What issues did you encounter?